### ISSUE TITLE
预测的时候batch_size只能设为1，并且用的是cpu

### ISSUE THREAD

**Issue by mmiku1** [struggler176393/Pointnet_book_seam#1]
您好，我在预测S3DIS数据集时batch_size只能设为1，请使用的是cpu，请问是否能调大batch_size，用gpu加速预测的速度呢，代码该如何调整呢，我注意到batch_size设置为1以外的数值时，  point_set_without_normal = np.asarray(torch.cat([points.permute(0, 2, 1), target[:, :, None]], dim=-1)).squeeze(0) 这部分代码会报错
